libs/anglify/src/lib/modules/radio-button/radio-button.module.ts

```diff
@@ -0,0 +1,12 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { RadioButtonComponent } from './radio-button.component';
+import { OverlayModule } from '../overlay/overlay.module';
+
+@NgModule({
+  declarations: [RadioButtonComponent],
+  imports: [CommonModule, FormsModule, OverlayModule],
+  exports: [RadioButtonComponent],
```

Comment:
Please export the OverlayModule here additionally

Reply:
Added OverlayModule to exports.